smurfs/src/components/SmurfList.js

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+
+
+import { getSmurfsData, deleteSmurf } from '../store/actions';
+import Smurf from './Smurf';
+
+const SmurfList = props => {
+
+  const getSmurfs = e => {
+    e.preventDefault();
+    props.getSmurfsData();
+  };
+
+
```

Comment:
you would probably add the delete button here as well as the logic that calls the action. 
this way you can directly grab the id off the smurf and have it get passed in to the function without having to sift through data. 
also add the smurf list here haha